public/app/plugins/datasource/influxdb/components/editor/query/influxql/visual/PartListSection.tsx

```diff
@@ -90,7 +70,7 @@ const Part = ({ name, params, onChange, onRemove }: PartProps): JSX.Element => {
   };
   return (
     <div className={partClass}>
-      <RemovableName name={name} onRemove={onRemove} />(
+      <button className={cx('gf-form-label', noRightMarginPaddingClass)}>{name}</button>
```

Comment:
```suggestion
      <button className={cx('gf-form-label', noRightMarginPaddingClass)}>{name}</button>(
```

Reply:
We need parenthesis. Without it it looks like this: 
![image](https://github.com/grafana/grafana/assets/820946/47fba73d-e111-4fce-a998-165121075fce)

Reply:
After this little change I think we are ready to merge.

Reply:
@itsmylife Thank you. I'm using vm, so I apologize for any inconvenience.

Reply:
No worries at all!